### PR TITLE
adding more unit tests

### PR DESCRIPTION
**Pull request by sbenedicadb**

### CODE REVIEW

**Review by rymorale — APPROVED** [review]
looks good to me 👍

File: AEPMessaging/Tests/UnitTests/MessagingEdgeEventTypeTests.swift [hunk 1]
```diff
@@ -12,17 +12,60 @@
...
-import AEPCore
 @testable import AEPMessaging
-import AEPServices
 
 class MessagingEdgeEventTypeTests: XCTestCase {
-    override func setUp() {
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+    func testInAppDismiss() throws {
+        // setup
+        let value = MessagingEdgeEventType(rawValue: 0)
```
Comment (rymorale) [inline AEPMessaging/Tests/UnitTests/MessagingEdgeEventTypeTests.swift:20]: nit: extra space after `//setup` here and the following test

**Review by rymorale — APPROVED** [review]

File: AEPMessaging/Tests/UnitTests/Messaging+PublicApiTest.swift [hunk 2]
```diff
@@ -119,4 +119,107 @@ class MessagingPublicApiTest: XCTestCase {
...
+        
+        EventHub.shared.getExtensionContainer(MockExtension.self)?.eventListeners.clear()
+        EventHub.shared.getExtensionContainer(MockExtension.self)?.registerListener(type: MessagingConstants.Event.EventType.messaging, source: EventSource.requestContent) { event in
+            XCTAssertEqual(MessagingConstants.Event.Name.REFRESH_MESSAGES, event.name)
+            XCTAssertEqual(MessagingConstants.Event.EventType.messaging, event.type)
+            XCTAssertEqual(EventSource.requestContent, event.source)
+            
+            guard let eventData = event.data else {
+                XCTFail()
+                expectation.fulfill()
```
Comment (PravinPK) [inline AEPMessaging/Tests/UnitTests/Messaging+PublicApiTest.swift:211]: can we remove this expectation.fulfill() ?

**Review by PravinPK — APPROVED** [review]

File: AEPMessaging/Tests/UnitTests/Messaging+PublicApiTest.swift (hunk 2 above)
Comment (sbenedicadb) [reply AEPMessaging/Tests/UnitTests/Messaging+PublicApiTest.swift:211]: it's a short-circuit on the expectation so we don't `wait` the full duration

**Pull request merged**